external/storm-kafka/src/jvm/storm/kafka/SpoutConfig.java

```diff
@@ -39,6 +39,8 @@
     public double retryDelayMultiplier = 1.0;
     public long retryDelayMaxMs = 60 * 1000;
```

Comment:
Since the above values only apply to the ExponentialBackoff implementation, it would be nice if we could factor them out of this class. They could probably just be moved to `ExponentialBackoffMsgRetryManagerFactory`, and in that case we could probably eliminate the factory pattern altogether.